src/components/AddOffering/AddOffering.js

```diff
@@ -11,22 +22,352 @@ const useStyles = (theme) => ({
     outline: "none",
     position: "relative",
   },
+  tabsRoot: {
+    marginBottom: "30px",
+    outline: "display",
+  },
+  modalHeader: {
+    paddingBottom: "20px",
+  },
+  submitBtn: {
+    backgroundColor: "#6ab547",
+  },
 });
 
+//-------------- Helper functions -----------------------//
+
```

Comment:
**Note to self:**
These two helpers will need to change to reflect the changes incoming from Authentication branch. Will adjust them when that branch is merged